seqr/views/utils/export_utils_2_3_tests.py

```diff
@@ -65,10 +67,13 @@ def mock_write_zip(file, content):
         })
         mock_zip_content = {}
 
+        # test tsv format with a filename in unicode
         response = export_multiple_files(
-            [['file1', header1, rows], ['file2', header2, rows]], 'zipfile', file_format='tsv')
+            [['file1', header1, rows], ['file2', header2, rows]], 'zipfilen\xe2me', file_format='tsv')
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.get('content-disposition'), 'attachment; filename="zipfile.zip"')
+        filename = response.get('content-disposition')
+        self.assertEqual(filename, 'attachment; filename="zipfilen\xc3\xa2me.zip"' if isinstance(filename, str)
+        else 'attachment; filename="zipfilen\xe2me.zip"'.encode('utf-8'))
```

Comment:
This is a really fragile looking test and I'm not comfortable with it. Moreover, I'm now concerned that if the header type is bytes and not a string it might not actually work the way we want in real life. I really think we should just strip out unicode characters and use plain strings unless you can get this to work with a single predictable header